src/components/Menu/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { Meta, Story } from '@storybook/react';
+
+import { Divider } from '@components/Divider';
+
+import { Menu as MenuComponent, MenuProps } from '../Menu';
+import { MenuItem } from '../components';
+import { COMPONENT } from './constants';
+
+export default {
+  title: COMPONENT.TITLE,
+  component: MenuComponent,
```

Comment:
MenuItem would be the SubComponent/Child of Menu component, Is it okay to not mention as subcomponent in this export? Just a doubt

Reply:
yes, good point. Added